src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -204,9 +204,16 @@ protected function dotize($id)
     /**
      * @internal
      */
-    protected function escape(string $string): string
+    protected function escape($value): string
     {
-        return addslashes($string);
+        if (\is_string($value)) {
+            $value = \addslashes($value);
+        }
+        if (\is_bool($value)) {
+            $value = $value ? '1' : '0';
+        }
+
+        return $value;
```

Comment:
this won't escape objects that implement __toString, I'd suggest to change this to

`return \is_bool($value) ? ($value ? '1' : '0') : addslashes($value);`

Reply:
Values passed to this function are only string and bool, you can check `attributes` array in same class.

But after checking it, I saw that we also can have integers passed here ... let me fix this !